description/film.rdfs.ttl

```diff
@@ -0,0 +1,67 @@
+@prefix dbo: <http://dbpedia.org/ontology/> .
+@prefix haFil: <https://data.hetarchief.be/ns/film/> .
+@prefix haObj: <https://data.hetarchief.be/ns/object/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+# Classes
+
+haFil:FilmType   a   rdfs:Class ;
+    rdfs:label "van het type Film"@nl ;
+    rdfs:label "of type Film"@en ;
+    rdfs:label "du genre Film"@fr ;
+    skos:definition """FilmType is van toepassing op media in meemoo's archief dat oorspronkelijk afkomstig is van een analoge drager van het type film."""@nl ;
+    skos:definition """FilmType applies to media in meemoo's archive that is originally derived from an analog carrier of type film."""@en ;
+    skos:definition """FilmType s'applique aux médias dans les archives de meemoo qui proviennent à l'origine d'un support analogue du type film."""@fr ;
+    rdfs:isDefinedBy <https://data.hetarchief.be/ns/film/> .
+
+# Properties                                                                            
+                                                                             
+haFil:numberAudioReelsLost a rdf:Property ;
+    rdfs:domain haObj:CarrierRepresentation ;
+    rdfs:range rdfs:Literal ;
+    rdfs:label "verloren audiospoelen"@nl ;
+    rdfs:label "lost audio reels"@en ;
+    rdfs:label "bobine audio perdues"@fr ;
+    skos:definition "Duidt aan voor een film of (en hoeveel) audiorollen/-spoelen verloren zijn, voorafgaand aan enig digitaliseringsproces."@nl;
+    skos:definition "Indicates for a film if (and how many) audio reels were lost, preceding any digitization process."@en;
+    skos:definition "Indique pour un film si (et combien) des bobines audio ont été perdues, avant toute opération de numérisation."@fr;
+    rdfs:isDefinedBy <https://data.hetarchief.be/ns/film/> .
+
+haFil:imageReelColoringType a rdf:Property ;
```

Comment:
(I also found this property: https://movielabs.com/cwontology/htmldoc/)

Reply:
I'll have to check about the need for separate Reel objects.

Regarding the property: I think the ebucore one has a different use ('colour space'/kleurruimte seems, to me, to be about a digital video vs. the physical colouring of a reel in this case).

The Movielabs ontology seems promising, but the link refers to a class, which can then use [this property](https://movielabs.com/cwontology/htmldoc/eidr_hasEditColorType.html), which then seems a bit more vague to me.

Reply:
The ontology was recently superseded by https://mc.movielabs.com/docs/ontology/, but that said: I don't think the semantics match that well. It seems they mean 'in the context of video editing', which is more aligned with wat EBUCore means.

A `haFilm:coloringType/coloringSpace` will probably suffice and the range can be `eidr:ColorType` or since the ontology is dead anyway: `haFilm:ColorType` with  Black and white, Color, Colorized, Composite, Unknown in a thesaurus.

The reels suggestion is interesting, but I'm not sure how it would be aligned. Would it be a subclass of PhysicalCarrier? It's probably a bit too expressive for now

Reply:
btw, you cannot replace the numberOfReels property this way because you never know if all reels are described. Creating 'dummy' reels/PhysicalCarriers' seems convoluted to me.

Reply:
I would place it on the same level as PhysicalCarrier; so something is both a PhysicalCarrier (in order to comply with the generic object model), and also either an AudioReel or an ImageReel (in order to comply with the descriptive film model).

e.g. 

```
<49c7e8fd18> a haObj:CarrierRepresentation ;
    schema:name "Dragerrepresentatie van 5q4rj6d538"@nl;
    rel:rep <5q4rj6d538> ;
    premis:storedAt <fddf731076>, <3ebec2adb1> ;
    haFil:numberOfReels "2"^^xsd:nonNegativeInteger .

<fddf731076> a haObj:PhysicalCarrier ;
    a haFil:ImageReel ;
    schema:name "Fysieke drager van 5q4rj6d538"@nl ;
    # Dynamic.barcode_image_reels
    schema:identifier "AFLM_KAD_003011"^^xsd:string;
    # dc_identifier_local_id
    premis:identifier [
        a haObj:LocalIdentifier ;
        rdf:value "BE/942855/111/39"^^xsd:string ;
    ];
    premis:medium haCarId:16mmfilm ;
    # or use a thesaurus for this
    schema:material "Original positive"^^xsd:string ;
    # or use a thesaurus for this
    haFil:coloringType "Color"^^xsd:string .

<3ebec2adb1> a haObj:PhysicalCarrier ;
    a haFil:AudioReel ;
    schema:name "Fysieke drager van 5q4rj6d538"@nl ;
    # Dynamic.barcode_sound_reels
    schema:identifier "AFLM_KAD_003012"^^xsd:string;
    # dc_identifier_local_id
    premis:identifier [
        a haObj:LocalIdentifier ;
        rdf:value "BE/942855/111/39"^^xsd:string ;
    ];
    premis:medium haCarId:openreelaudiotape .
```

Reply:
@milvld this is actually perfectly fine! subclass relation is not necessay and maybe even undesired. This is the way to go!

Reply:
And maybe to add to that:
-  maybe have a haFil:Reel that is the superclass of ImageReel and AudioReel (is SoundReel more common?) for completion
- haFil:coloringType is a property of haFil:ImageReel I presume?
- I think it's ok to make this string or maybe have a `sh:in` like https://github.com/viaacode/datamodels/blob/da09672b74fca2095956bbebd472b34b949c9c8e/events/events.shacl.ttl#L144